addons/hr/migrations/12.0.1.1/openupgrade_analysis.txt

```diff
@@ -27,5 +27,6 @@ hr           / mail.channel             / subscription_department_ids (many2many
 ---XML records in module 'hr'---
 NEW hr.employee: hr.employee_admin (noupdate)
 DEL hr.employee: hr.employee_root (noupdate)
+NEW ir.rule: hr.hr_employee_comp_rule (noupdate)
```

Comment:
Does this changes the behavior of previous version? We may need to disable it when migrating.

Reply:
No problem. In v11, hr.employee didn't have a company_id field.

Reply:
Well, then there's a problem: all companies see all employees, and now there are not going to do it, so we have to query if several companies, and if so, deactivate the rule. And also be sure the field remains empty.

Reply:
EDIT: In v11, `hr.employee` has `company_id` because does inherit of `resource.mixin`. Thus means, in employees have the company of their resource.

Reply:
OK, we can leave it this way then.